src/main/java/to/us/resume_builder/ResumeExporter.java

```diff
@@ -45,20 +45,33 @@ public static String uploadPDF(Path pdf) throws IOException, InterruptedExceptio
 
         // Create the command
         PostRequest pr = new PostRequest("/", pdf.toString());
-        LOG.info("Attempting to post");
+        LOG.info("Attempting to POST to " + ApplicationConfiguration.getInstance().getString("upload.url"));
 
         try {
-            HttpResponse<InputStream> result = pr.sendRequest("expires", "2w");
+            HttpResponse<InputStream> result;
+            if (ApplicationConfiguration.getInstance().getString("upload.url").equals("file.io")) {
+                result = pr.sendRequest("expires", "2w");
+            } else {
+                result = pr.sendRequest();
+            }
             String response = new String(result.body().readAllBytes(), StandardCharsets.UTF_8);
-            if (HttpStatus.resolve(result.statusCode()).is2xxSuccessful()) {
+            boolean success;
+            if (Objects.requireNonNull(HttpStatus.resolve(result.statusCode())).is2xxSuccessful()) {
                 LOG.info("Response: " + response);
+                success = true;
             } else {
                 LOG.warning("Failed request: " + response);
+                success = false;
+            }
+
+            if (ApplicationConfiguration.getInstance().getString("upload.url").equals("transfer.sh")) {
+                response = "{\"success\":" + success + ",\"link\":\"" + response.replace("transfer.sh/", "transfer.sh/download/") + "\",\"expiry\":\"14 days\"}";
+                LOG.info("Updated response: " + response);
             }
 
             return response;
         } catch (HttpTimeoutException ex) {
-            LOG.warning("Upload to file.io timed out");
+            LOG.warning("Upload to transfer.sh timed out");
```

Comment:
Isn't the site you upload to customizable now? Shouldn't it state which site the upload failed for, instead of assuming one or the other?